src/foam/graphics/TreeGraph.js

```diff
@@ -115,7 +115,8 @@
            this.invalidate();
            this.doLayout();
 
-           this.onSizeChange.pub();
+         } else {
+          this.onSizeChangeComplete.pub();
```

Comment:
Who is listening to this?

Reply:
The AccountTreeView search feature on when to scroll. Basically the search feature only should scroll once the canvas has finished changing size and the nodes have been fully laid out which wasn't the case before, the scroll was triggering only when the canvas changes.